lab-davide/app/html/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp"> <!--ng angular directive. uses this tag as the basis for html-->
+  <head>
+    <meta charset="utf-8">
+    <title>What do Cows actually say?</title>
+    <link rel="stylesheet" type= "text/css" href="bundle.css"/>
+    <script src ="bundle.js"></script>
+  </head>
+  <body>
+    <main ng-controller = "cowsayController">
+      <form ng-submit = "done()">
+      <label for ="cowsay-text">What say COWSAY??</label>
+        <input
+          name: "cowsay text"
+          type: "text"
+          ng-model = "cowsayText" />
+          <select ng-model = "cowsayFile">
+            <option ng-repeat = "cow in cowFiles" value= "{{cow}}">{{cow}}</option>
+          </select>
+          <button type = "submit">MOO-DONE</button>
+      </form>
+      <p>preview</p>
+      <pre> {{ say() }} </pre>
```

Comment:
I would change functions to be named stuff that makes sense to you. If 'say' works for you, that's fine, but I would have called this something like 'runInputTextThroughCowsay' or something.